bundles/com.amazonaws.eclipse.core/src/com/amazonaws/eclipse/core/mobileanalytics/context/ClientContextConfig.java

```diff
@@ -137,7 +169,7 @@ private static String _getSystemLocaleCountry() {
         }
     }
 
-    private static String _getOrGenerateClientId() {
+    public static String _getOrGenerateClientId() {
```

Comment:
Now that it's public, remove the underscore from the name

Reply:
I want to do a pr that removes all the underscore from private, second worst old naming thing next to hungarian notation, fixed